components/prism-cmake.js

```diff
@@ -0,0 +1,17 @@
+Prism.languages.cmake = {
+    'comment': {
+	  pattern: /#.*/,
```

Comment:
You seem to have mixed indentation here.

Reply:
Also, you can inline the regex to get

```js
'comment': /#.*/,
...
```

Reply:
Sorry about that, I think I have fixed them properly. However, I might still need to convert it to tabs. Isn't there some automatic an formatter like clang-format in C++?

Reply:
I'm using VSCode. It can auto convert back and forth between spaces and tabs.

If you don't find any better solution, a replace regex usually works wonders.

Reply:
ok

Reply:
Fixed.